Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -501,7 +501,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        return empty($changed) ? 0 : 1;
+        return !$resolver->isDryRun() || empty($changed) ? 0 : 2;
```

Comment:
exit code 2 is reserved for `Misuse of shell builtins (according to Bash documentation)`